scene/gui/graph_edit.cpp

```diff
@@ -1322,18 +1323,10 @@ void GraphEdit::_gui_input(const Ref<InputEvent> &p_ev) {
 			minimap->update();
 		}
 
-		if (b->get_button_index() == MOUSE_BUTTON_WHEEL_UP && Input::get_singleton()->is_key_pressed(KEY_CONTROL)) {
+		if (b->get_button_index() == MOUSE_BUTTON_WHEEL_UP) {
 			set_zoom_custom(zoom * ZOOM_SCALE, b->get_position());
-		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_DOWN && Input::get_singleton()->is_key_pressed(KEY_CONTROL)) {
+		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_DOWN) {
 			set_zoom_custom(zoom / ZOOM_SCALE, b->get_position());
-		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_UP && !Input::get_singleton()->is_key_pressed(KEY_SHIFT)) {
-			v_scroll->set_value(v_scroll->get_value() - v_scroll->get_page() * b->get_factor() / 8);
-		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_DOWN && !Input::get_singleton()->is_key_pressed(KEY_SHIFT)) {
-			v_scroll->set_value(v_scroll->get_value() + v_scroll->get_page() * b->get_factor() / 8);
-		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_RIGHT || (b->get_button_index() == MOUSE_BUTTON_WHEEL_DOWN && Input::get_singleton()->is_key_pressed(KEY_SHIFT))) {
-			h_scroll->set_value(h_scroll->get_value() + h_scroll->get_page() * b->get_factor() / 8);
-		} else if (b->get_button_index() == MOUSE_BUTTON_WHEEL_LEFT || (b->get_button_index() == MOUSE_BUTTON_WHEEL_UP && Input::get_singleton()->is_key_pressed(KEY_SHIFT))) {
-			h_scroll->set_value(h_scroll->get_value() - h_scroll->get_page() * b->get_factor() / 8);
```

Comment:
I am fine with removing the Control modifier requirement (and am of the opinion it is the right way to go), but some people won't be - there is no winner to this battle.

What I don't understand though is removing all the vertical and horizontal scrolling functionality? That is legitimately useful, why remove it? In this case I think the Blender node editor serves as a good example:
```
Scroll = Zoom
Ctrl + Scroll = Horizontal Pan
Shift + Scroll = Vertical Pan
```

Reply:
I didn't know Blender did that, that's an interesting idea, I'll do that

Reply:
I made it
```
Ctrl + Scroll = Vertical Pan
Shift + Scroll = Horizontal Pan
```
(See https://chat.godotengine.org/channel/editor?msg=9t53ykBiAAJtzpA2s)